commitlint.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  extends: ["gitmoji"],
+  parserPreset: {
+      parserOpts: {
```

Comment:
@tehKapa is this compatible with the commit style you proposed?

Reply:
theoretically, yes. however, I also extends this `['@commitlint/config-conventional']`
@PhearZero did you use both solutions in your tests?

Reply:
I really like conventional. In addition to it we need also to enforce referencing the issue the commits are addressing.

Reply:
I started with config-conventional linting for testing locally and then added in alternative configurations in the PoC testing pnpm/husky.  We could rewrite the history if need be

@fox1t @tehKapa

Reply:
Is there anything that I need to do here?

This looks like it matches the config in the [hospitalrun/hospitalrun](https://github.com/HospitalRun/hospitalrun/blob/master/commitlint.config.js) repository.

If so, I would assume we would update this config to match [hospitalrun/components](https://github.com/HospitalRun/components/blob/master/commitlint.config.js)